openmetadata-ui/src/main/resources/ui/src/components/Explore/EntitySummaryPanel/SearchIndexSummary/SearchIndexSummary.test.tsx

```diff
@@ -33,7 +33,9 @@ describe('SearchIndexSummary component tests', () => {
   it('Component should render properly, when loaded in the Explore page.', async () => {
     await act(async () => {
       render(
-        <SearchIndexSummary entityDetails={mockSearchIndexEntityDetails} />
+        <BrowserRouter>
```

Comment:
We can have MemoryRouter as wrapper instead manually adding BrowserRouter check other specs for ref